src/main/java/se/isselab/HAnS/pluginExtensions/backgroundTasks/featureTasks/GetFeatureMetricsForFeature.java

```diff
@@ -0,0 +1,43 @@
+package se.isselab.HAnS.pluginExtensions.backgroundTasks.featureTasks;
+
+import com.intellij.openapi.progress.ProgressIndicator;
+import com.intellij.openapi.progress.Task;
+import com.intellij.openapi.project.Project;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import se.isselab.HAnS.featureModel.psi.FeatureModelFeature;
+
+public class GetFeatureMetricsForFeature extends Task.Backgroundable{
```

Comment:
Could also make this more efficient by not getting the featurefilemappings over again for the subtasks and rather pass it on.